src/api/integration-requests.md

```diff
@@ -25,18 +25,18 @@ Integration Requests enable terminal integrations and acceptance of asset types
 |   Field    |  Type  |                                                           Description                                                            |
 | :--------- | :----- | :------------------------------------------------------------------------------------------------------------------------------- |
 | id         | String | The Integration Request's unique identifier.                                                                                     |
-| accountId  | String | The Integration Request's owning Centrapay Account id.                                                                           |
```

Comment:
AccountId is still a mandatory field of an integration request. This PR should only remove the accountId coming from the request body

Reply:
I have reverted the model to how it was